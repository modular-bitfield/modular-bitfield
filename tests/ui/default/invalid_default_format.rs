use modular_bitfield::prelude::*;

#[bitfield]
pub struct InvalidDefaultFormat {
    #[default]
    pub field: bool,
    pub other: B7,
}

fn main() {}