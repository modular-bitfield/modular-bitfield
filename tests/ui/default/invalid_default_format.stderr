error: encountered invalid format for #[default] field attribute, expected #[default = expression]
 --> tests/ui/default/invalid_default_format.rs:5:7
  |
5 |     #[default]
  |       ^^^^^^^
